src/Validation/Validator.php

```diff
@@ -1320,6 +1420,14 @@ public function equals(
         ?string $message = null,
         Closure|string|null $when = null
     ) {
+        if ($message === null) {
+            if (!$this->_useI18n) {
+                $message = sprintf('The provided value must be equals to `%s`', $value);
+            } else {
+                $message = __d('cake', 'The provided value must be equals to `{0}`', $value);
```

Comment:
```suggestion
                $message = sprintf('The provided value must be equal to `%s`', $value);
            } else {
                $message = __d('cake', 'The provided value must be equal to `{0}`', $value);
```

https://www.quora.com/Which-is-correct-is-equals-to-OR-is-equal-to